modules/aws_k8s_base/aws-k8s-base.json

```diff
@@ -61,6 +61,21 @@
       "description": "Expose self-signed SSL certs.",
       "default": false
     },
+    "cert_manager_values": {
+      "type": "object",
+      "description": "Certificate Manager helm chart additional values. [Available options](https://artifacthub.io/packages/helm/cert-manager/cert-manager?modal=values)",
+      "default": {}
+    },
```

Comment:
let's not change cert manager since we plan to delete it

Reply:
I'll refactor this in another PR.